Source/ActionController.swift

```diff
@@ -179,13 +179,29 @@ open class ActionController<ActionViewType: UICollectionViewCell, ActionDataType
             section.actions.append(action)
         }
     }
-
     @discardableResult
     open func addSection(_ section: Section<ActionDataType, SectionHeaderDataType>) -> Section<ActionDataType, SectionHeaderDataType> {
         _sections.append(section)
         return section
     }
     
+    public typealias ClosureAction = ((Action<ActionDataType>) -> Bool)
+    
+    open func removeAction(closure: ClosureAction) {
+        if let section = _sections.last {
```

Comment:
here you need to iterate through all the sections available and for each one through all its actions.